XRDXRFutils/spectra.py

```diff
@@ -240,26 +231,30 @@ class FastSpectraXRD():
 
     def __init__(self):
         pass
+    
 
-    def fromDataf(self, data, i):
+    def from_Data(self, data, x, y):
 
         self.opt = data.opt.copy()
 
-        self.counts = data.data.reshape(-1,1280)[i]
```

Comment:
I prefer this as I always know how the array is unrolled.

Reply:
Ok, now I take `from_Dataf()` as base and write `from_Data()` that calls `from_Dataf()`.